### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ogiogiovictor**
## Project requirements
- Add RSpec as a gem to your project.
-  Create a Request spec file for all your controllers.
- Make sure that for each action you check:
- If response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    it 'returns http success' do
+      get '/users/'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'Renders the right template index' do
+      get '/'
+      expect(response).to render_template(:index)
+    end
+  end
```
Comment (mikethreels) [inline spec/requests/users_spec.rb:15]: - [ ] both the index and the show actions should be tested in this test file

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    it 'Render the right index template for post' do
+      expect(response).to render_template(:index)
+    end
+
+    describe 'Get all related routes to post show template' do
+      before :each do
+        get '/users/:user_id/posts/id'
+      end
+    end
```
Comment (mikethreels) [inline spec/requests/posts_spec.rb]: - [ ] both the index and the show actions should be tested in this test file

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Victor, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @ogiogiovictor, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**